blocks/library/image/index.js

```diff
@@ -92,6 +92,8 @@ registerBlock( 'core/image', {
 			);
 		}
 
+		const focusCaption = ( focusValue = {} ) => setFocus( { editable: 'caption', ...focusValue } );
```

Comment:
Object spread (or more accurately, `Object.assign`) will allow undefined values as long as it's not the first argument, so technically the `= {}` default isn't necessary.